dhall-json/src/Dhall/JSON.hs

```diff
@@ -107,15 +107,11 @@
 
 >     let Example = < Left : { foo : Natural } | Right : { bar : Bool } >
 > 
-> in  let example = constructors Example
-> 
 > in  let Nesting = < Inline : {} | Nested : Text >
 > 
-> in  let nesting = constructors Nesting
-> 
 > in  { field    = "name"
->     , nesting  = nesting.Inline {=}
->     , contents = example.Left { foo = 2 }
+>     , nesting  = Nesting.Inline {=}
+>     , contents = Example.Left { foo = 2 }
 >     }
 
     ... produces this JSON:
```

Comment:
This example seems to have bit-rotted. After my change I get

```
cat example.dhall | dhall-to-json 
{"field":"name","contents":{"foo":2},"nesting":{}}
```

Reply:
I have a fix up for those in https://github.com/dhall-lang/dhall-haskell/pull/1080